faqs/refunds-faq.md

```diff
@@ -36,6 +42,56 @@ All integrations with the Vipps eCom API _must_  support refunds.
 It is also possible to do refunds on
 [portal.vipps.no](https://portal.vipps.no).
 
-See
-[Refunds](https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api-faq.md#refunds)
-in the eCom FAQs.
+
+Refunds can be made up to 365 days after payment or reservation.
+Very old payments have a higher risk of being problematic, because people
+change banks, leave the country, die, etc,
+and this then requires time-consuming manual work.
+Vipps therefore limits refunds to 365 days.
+
+### How can I refund only a part of a payment?
+
+Example: A customer has placed an order of of two items for a total of 1000 NOK.
+The merchant has initiated a payment of 1000 NOK, but the customer has changed
+her mind and only bought one of the items, with a price of 750 NOK. The merchant
+has therefore made a
+[partial capture](../common-topics/reserve-and-capture#partial-capture)
+of 750 NOK, and need to refund the remaining 250 NOK.
+
+* The short version: This is done automatically by the bank after a few days.
+See:
+[For how long is a payment reserved?](reserve-and-capture-faq.md#for-how-long-is-a-payment-reserved).
+
+* The long version: It _is_ possible to cancel the remaining reservation after a
+partial capture through Vipps: Send a
+[`PUT:/ecomm/v2/payments/{orderId}/cancel`](https://vippsas.github.io/vipps-developer-docs/api/ecom#tag/Vipps-eCom-API/operation/cancelPaymentRequestUsingPUT)
+request with `shouldReleaseRemainingFunds: true` in the body.
+The payment must be `RESERVED` for this to take effect.
+See:
+[Cancelling a partially captured order](https://vippsas.github.io/vipps-developer-docs/docs/APIs/ecom-api/vipps-ecom-api#cancelling-a-partially-captured-order).
```

Comment:
I'm not sure about this, but should we indicate that the link is to the eCom API, and not general info for all APIs?

```suggestion
[Vipps eCom API: Cancelling a partially captured order](https://vippsas.github.io/vipps-developer-docs/docs/APIs/ecom-api/vipps-ecom-api#cancelling-a-partially-captured-order).
```